src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/opt.cpp

```diff
@@ -425,10 +472,6 @@ DQParMMGQ::DQParMMGQ(Context::Ref ctx) {
 }
 
 void mergeParallelMatMuls(const std::shared_ptr<ov::Model>& m, Context& ctx) {
-    ov::pass::GraphRewrite rewr;
-    rewr.add_matcher<ov::npuw::patterns::opt::DQParMMGQ>(std::ref(ctx));
```

Comment:
Does it still work for the repeated block?

Reply:
It does, why it shouldn't? Though this PR reorganized the order for the passes, this DQParMMGQ is still called in `optimize()` - https://github.com/openvinotoolkit/openvino/pull/26633/files#diff-74c8442fb8280a82e1304119bbf6c1ce6ad4db3980d16b24bef9aa0c41f5fc3aR1603